molecule/default/tests/test_verify_mbu_installed.py

```diff
@@ -10,15 +16,41 @@
 backup_folder_on_container = '/var/backup'
 
 
-@pytest.mark.incremental
+# ======================================================================================================================
+# Helpers
+# ======================================================================================================================
+def run_on_container(command, container, run_on_host):
```

Comment:
Should this helper be removed and the one from https://github.com/rcbops/pytest-rpc/blob/master/pytest_rpc/helpers.py#L510-L527 ?